CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 ## [1.X.X] - 20XX-XX-XX
 
 ### Added
+- power-to-fuel system, power-to-fuel process, power-to-ammonia system, power-to-liquid process (#1477)
```

Comment:
Minor thing: We not here the PR number (1483) and not this issue number (1477). Also we sort by PR number, so it should be after the line
`- electricity cost (#1482)`